reference/5.1/Microsoft.PowerShell.Management/Get-ChildItem.md

```diff
@@ -2,7 +2,7 @@
 external help file: Microsoft.PowerShell.Commands.Management.dll-Help.xml
 Locale: en-US
 Module Name: Microsoft.PowerShell.Management
-ms.date: 11/13/2023
+ms.date: 03/28/2024

```

Comment:
According to the [Editor's checklist](https://learn.microsoft.com/en-us/powershell/scripting/community/contributing/editorial-checklist?view=powershell-7.4#metadata), maybe I shouldn't update `ms.date`.

> - `ms.date`: must be in the format **MM/DD/YYYY**
>   - Change the date when there is a significant or factual update
>     - Reorganizing the article
>     - Fixing factual errors
>     - Adding new information
>   - Don't change the date if the update is insignificant
>     - Fixing typos and formatting